spec/intl.html

```diff
@@ -502,7 +504,7 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
       <emu-alg>
         1. <ins>Let _x_ be ? HandleDateTimeValue(_dateTimeFormat_, _x_).</ins>
         1. Let _patternParts_ be PartitionPattern(<del>_dateTimeFormat_.[[Pattern]]</del><ins>_x_.[[pattern]]</ins>).
-        1. Let _result_ be ? FormatDateTimePattern(_dateTimeFormat_, <ins>_x_.[[pattern]]</ins>, _patternParts_, <del>_x_</del><ins>_x_.[[epochNanoseconds]]</ins>, *undefined*).
+        1. Let _result_ be ? FormatDateTimePattern(_dateTimeFormat_, <ins>_x_.[[pattern]]</ins>, _patternParts_, <del>_x_</del><ins>_x_.[[epochNanoseconds]]</ins>, *undefined*, <ins>x.[[timeZone]]</ins>).
```

Comment:
notice the only possible value for _x_.[[timeZone]] here is either *undefined* or the same value of DefaultTimeZone() but nothing else. The value DefaultTimeZone() came from ZDT and the value *undefined* came from other data type. There is impossible to have any other values at this point Jan 31, 2023  655e3ef .